routes/categoryRouter.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const router = express.Router();
+
+const categoryController = require("../controllers/categoryController");
```

Comment:
```suggestion
const express = require("express");

const categoryController = require("../controllers/categoryController");

const router = express.Router();
```
모듈화 시켜놓은 file 내에서 코드를 작성하는 순서는 필요한 module을 convention에 맞게 (보통 built-in module -> 외부 모듈 -> custom 모듈 순으로 작성) 불러오고 이를 활용하는 순서로 작성되는게 일반적이고 읽기 좋습니다 :) 현재 작성해주신 코드로도 동작에는 이상이 없으나 suggestion과 함께 위 내용을 참고해주세요~